Lab 2/token.in

```diff
@@ -0,0 +1,29 @@
++
+-
+*
+/
+%
+<
+<=
+=
+>=
+==
+[
+]
+:
+SPACE
+NEWLINE
+TAB
+<<
+>
```

Comment:
The file lacks the ">>" operator for the read operation. Also, I think it would make more sense if ">" was moved near the relation operators (lines 6-10) and ">>" was added here.

Reply:
Good idea! Thank you!